ConfigApp/MainWindow.xaml.cs

```diff
@@ -411,6 +411,10 @@ private void user_save_Click(object sender, RoutedEventArgs e)
             WriteTwitchFile();
             WriteEffectsFile();
 
+            // Reload saved config to show the "new" (saved) settings
+            ParseConfigFile();
+            ParseTwitchFile();

```

Comment:
This shouldn't be necessary since it fetches all the user inputs to write the config files and doesn't touch them in any way (so the user inputs should correspond to what's in the config files anyways).

Reply:
it DOES change it though, if you empty a textbox (timer duration for example) and saving it. thats what i noticed :)

Reply:
Yeah it doesn't update the text boxes if an user enters invalid input and saves it currently. Makes sense to read the config files again to update the views in those cases.